bigquery_etl/glam/templates/clients_daily_histogram_aggregates_v1.sql

```diff
@@ -23,20 +14,9 @@ extracted AS (
     client_info.app_channel AS channel
   FROM
     `moz-fx-data-shared-prod.{{ source_table }}`
-  {% if filter_desktop_builds %}
-  INNER JOIN
-    valid_builds
-  ON
-    client_info.app_build = build_id
-  {% endif %}
   WHERE
     DATE(submission_timestamp) = {{ submission_date }}
     AND client_info.client_id IS NOT NULL
-    {% if filter_desktop_builds %}
-    -- some FOG clients on Windows are sending version "1024.0.0"
-    -- see https://bugzilla.mozilla.org/show_bug.cgi?id=1768187
-    AND client_info.app_display_version != '1024.0.0'
```

Comment:
I'm not sure if removing this check is a good idea, but can be convinced otherwise

Reply:
In theory it's impossible to run into this problem again considering we now have [this guardrail that makes sure we only let in data with the correct latest version](https://github.com/mozilla/bigquery-etl/blob/main/bigquery_etl/glam/templates/latest_versions_v1.sql). We had a problem of version 999 before and it was correctly filtered out thanks to that step.

Reply:
> In theory it's impossible to run into this problem again considering we now have [this guardrail that makes sure we only let in data with the correct latest version](https://github.com/mozilla/bigquery-etl/blob/main/bigquery_etl/glam/templates/latest_versions_v1.sql). We had a problem of version 999 before and it was correctly filtered out thanks to that step.

I think this is correct.

Reply:
ok I've been convinced otherwise